mos-platform/atari2600-3e/mapper_3e.c

```diff
@@ -8,24 +8,24 @@
 
 __attribute__((section(".zp.bss"))) bank_index_t __current_bank;
 
-__attribute__((weak)) void set_current_bank(bank_index_t bank_id) {
-  if (bank_select && !(bank_id & 0x80)) {
-    bank_select(bank_id);
-  } else if (ram_select) {
+__attribute__((leaf)) void set_current_bank(bank_index_t bank_id) {
```

Comment:
To clarify; the basic rule is that no function definition in C should ever have `__attribute ((leaf))`. This is only useful for declarations of functions written assembly files, or much less commonly, C files that were intentionally compiled with `-fno-lto`.